test/Prelude.purs

```diff
@@ -106,7 +106,9 @@ check checkers execResult = do
       Left err -> err.stderr
       Right res -> res.stderr
 
-  when false do
+  printStdoutStderr <- liftEffect $ map isJust $ Process.lookupEnv "SPT_DEBUG" -- spago test
+
+  when printStdoutStderr do
```

Comment:
There's too many times I've wanted to enable this. So, I thought it'd be better if we used an env flag to do this. I chose `SPT_DEBUG` because it was shorter than `SPAGO_TEST_DEBUG`.

Reply:
Ah yes, brilliant. But let's go for `SPAGO_TEST_DEBUG` as "SPT" is a lil cryptic 😄 

This is also worth noting in the CONTRIBUTORS file

Reply:
Fixed.